### ISSUE TITLE
Micro Stutters in Monster Hunter Titles

### ISSUE THREAD

**Issue by MrMadness1** [citra-emu/citra#6484]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Nightly 1882/Canary 2468

### Description of Issue

Micro stuttering occurs in MH3U in most of the hub areas. Moga village and the taverns always have stuttering regardless of having texture packs enable or disabled. In missions the stutters seem to stop, unless in multiplayer, and primarily only occurs with 3 or more players. MHXX has these stutter issues occurring in game regardless of hub area or in mission. This does not seem to be a shader cache issue either as it continues to occur even after extending stay in 1 area.

### Expected Behavior

There shouldn't be micro stuttering.

### Reproduction Steps

Simply playing the game in either full screen or windowed mode.

### Log File

[citra_log.txt](https://github.com/citra-emu/citra/files/11360005/citra_log.txt)

### System Configuration

CPU: Ryzen 5 3600
GPU/Driver: RTX 3060ti
RAM: 16 gigs 3600 mhz
OS: Windows 10

**Comment by MrMadness1** [comment]
I've just tested nightly 1894 as well, same issue.

**Comment by DonKatsu** [comment]
Updated my Citra appimage from nightly 1500something to 2053 after not using it for quite a while.

MH4U still seems fine for me with OpenGL or Vulkan, but I did notice Vulkan's game speed keeps swaying between 99% and 101% gamespeed where OpenGL is a solid 100%. Had Render Settings (dithering and stuff) and Blur Removal cheats on.

Noticed MHXX stuttering running around Bherna Village, and in a quick test of a hunt. Drops of 5 FPS or so every so often. Had Unlock FPS and Render Settings cheats on.

MH3U had drops to 45 or so running around Moga Village. Some sizable drops in hunts as well. Had the 60 FPS cap cheat on.

Tried clearing caches and changing various settings, but nothing seemed to really help MH3U or MHXX.

CPU: 5900X
GPU: 6700XT
RAM: 32GB 3600 DDR4
OS: Fedora 39 KDE

**Comment by MrMadness1** [comment]
To date, micro stutters when using corrective FPS codes in 3U continue to occur. 4U seems to be the most stable at 60fps, and MHXX cannot go above 52fps without encountering very strong stuttering back to that 52fps mark. I play 3U at 30, 4U at 60, and XX at 50 fps for the most stable experience but would still like to see 3U and XX improved.

**Comment by MrMadness1** [comment]
Yep its happening for me at regular intervals. 3U's wonderful spaghetti code and its issues with framerate I'm sure isn't helping. For XX, I'm not sure what causes it but if you hard lock to 60, you can watch consistent drops to 52fps. I've tested this on about 10 different users devices at this point.
